### ISSUE TITLE
Sensitivity Settings and Macros

### ISSUE THREAD

**Issue by tremendoza** [Ryochan7/DS4Windows#655]
Adjusting axis sensitivity values seems to break key repeat for all buttons mapped to keyboard output, this issue has been present in earlier versions as well.

Macros that use combinations of the 'Alt' key and 'Arrow' keys now send wacky characters like hearts and diamonds instead of its intended output.

v1.7.2

**Comment by mika-n** [comment]
Take a look at the end of the following discussion: https://github.com/Ryochan7/DS4Windows/issues/654

Sensitivity calculation was a bit wrong but it is fixed in the current development version and the next release should have that fix. For now sensitivity should be set to 1.00 in all axies or the result is not exactly what it was supposed to be.

**Comment by tremendoza** [comment]
I don't have any sensitivity options adjusted because I noticed issues about 4 versions ago so the macro issue might be something else. Alt+up, down, left or right send text symbols instead of their navigation functions. Every other macro I have set works. Alt+tab, enter, f4, f1, f10 etc..

**Comment by mika-n** [comment]
Would you please share your profile file and macro definition file (actions.xml)? I just tested to record a macro where I used also ALT+UpArrow, ALT+DownArrow key combinations. Worked just fine.

**Comment by tremendoza** [comment]
[Navigation.zip](https://github.com/Ryochan7/DS4Windows/files/3075148/Navigation.zip)
As requested..

**Comment by Ryochan7** [comment]
This seems to be an issue when using the Scan Code option. That option in the whole program was deprecated early on. I have noticed that the Jays2Kings build behaves the same way so I don't think this would have worked at any point unless you had the option disabled which is the default. I have also checked the resulting output codes and the arrows are interpreted wrong. The arrows are being interpreted as numpad keys rather than the dedicated arrow keys. That is causing symbol generation to occur.

https://www.alt-codes.net/

You don't need the Scan Code (only) option. The normal key routine will output both the virtual key and the applicable scan code.

**Comment by tremendoza** [comment]
"The arrows are being interpreted as numpad keys rather than the dedicated arrow keys" got me thinking about numlock so I turned it on and the macros work again. I have no physical keyboard attached so I didn't realize it was off. Regarding the scan code option, I also have one of my macro trigger buttons mapped to the shift key; using the scan code option is the only way I found to still trigger the macro even though shift is also pressed. Since there isn't an option for multiple 'shift key' mappings per button or a way to ignore/block initial button triggers on a macro event.
